tests/infer/test_inference.py

```diff
@@ -47,12 +47,18 @@ def setUp(self):
         self.batch_size = 4
 
     def test_elbo_reparameterized(self):
-        self.do_elbo_test(True, 5000)
+        self.do_elbo_test(True, 5000, Trace_ELBO())
```

Comment:
For these tests, the value of the loss seems to be somewhat different (almost 3X or so) the value from `Trace_ELBO` with `alpha=0.99` (or any other value for that matter). Is that expected?

Reply:
If num_particles = 1, the loss would be the same.

Reply:
Sorry, I should have mentioned - with `num_particles=1`, in `NormalNormalTest`, the elbo loss returned by `svi.step` is around 6 for `test_elbo_reparameterized` and around 20 for `test_renyi_reparameterized` (and doesn't seem to change too much with `num_particles`).